src/components/wizard/wizard.component.ts

```diff
@@ -250,15 +250,15 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
     async finish(): Promise<void> {
 
         // fires when the finish button is clicked always
-        this.onFinishing.next();
+        this.onFinishing.emit();
 
         // Disable the button while waiting on validation
         this.finishDisabled = true;
 
         try {
             // Fetch validation status
             const validationResult = this.isStepValid();
-            this.getCurrentStep().valid = validationResult instanceof Promise ? await validationResult : validationResult;
+            this.getCurrentStep()._valid = validationResult instanceof Promise ? await validationResult : validationResult;
```

Comment:
Should this be calling the `setValid` function? I know there is logic involved in setting the valid state, just wondering if that logic needs to be run when this change is made? 

Also, I don't know for sure, but I imagine this could change the valid state, in which case we would need to emit the latest value to the user as well if it has changed?

Reply:
Did we determine if we needed to emit validChange here?